docs/source/conf.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import os
+from datetime import datetime
+sys.path.insert(0, os.path.abspath('../../'))
+
+rootdir = os.path.join(os.getenv("SPHINX_MULTIVERSION_SOURCEDIR", default=os.getcwd()), "..")
+sys.path.insert(0, rootdir)
+sys.path.append(os.path.abspath('../../python/src'))
+print("*****************************************")
+[print(p) for p in sys.path]
+print("*****************************************")
+
+import opendp.smartnoise
+import opendp.smartnoise.synthesizers
+print(opendp.smartnoise.__spec__)
+print("*****************************************")
+
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.extlinks',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.ifconfig',
+    'sphinx.ext.viewcode',
+    'sphinx.ext.graphviz',
+    'sphinx.ext.todo',
+    'sphinx_multiversion',
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix of source filenames.
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'OpenDP'
+copyright = u'%d' % datetime.now().year
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = '0.0.1'
```

Comment:
I was attempting to add some global checks to make sure version numbers across projects are synchronized in the CI release PR. There's a root-level VERSION file and a `.github/assert_version.py` that checks version numbers. I'm not sure if that's how we want to handle things, but if it is, maybe this could read the version from the VERSION file?

Reply:
Yes, I absolutely want us to have a single source of truth for the version. I just threw 0.0.1 in there for discussion. I'm glad you found it. Reading from a file should work, I would imagine.